tests/metrics/test_metric_definitions.py

```diff
@@ -266,3 +270,69 @@ def test_recall_at_k():
         "p90": 0.8,
         "variance": 0.1,
     }
+
+
+def test_ndcg_at_k():
+    # normal cases
```

Comment:
nit: I think that some of these cases are covered within `test_default_evaluator.py` already, so it could be worthwhile to trim down on these tests to make it easier to maintain in the future?

Reply:
Discussed offline: for future custom metrics, the correctness of the calculation is preferred to be written in `test_metric_definitions`. `test_default_evaluator` is supposed to make sure the built-in metrics are correctly associated with the corresponding model type.